apps/ledger-live-desktop/src/newArch/features/WalletSync/hooks/useLedgerSyncInfo.ts

```diff
@@ -0,0 +1,32 @@
+import { useQueries, UseQueryResult } from "@tanstack/react-query";
+import { QueryKey } from "./type.hooks";
+import TrustchainApi, { StatusAPIResponse as TrustchainStatus } from "@ledgerhq/trustchain/api";
+import CloudSyncApi, {
+  StatusAPIResponse as CloudSyncStatus,
+} from "@ledgerhq/live-wallet/cloudsync/api";
+
+export function useLedgerSyncInfo() {
+  const QUERIES = [
+    {
+      queryKey: [QueryKey.fetchTrustchainStatus],
+      queryFn: () => TrustchainApi.fetchStatus(),
+    },
+    {
+      queryKey: [QueryKey.fetchCloudSyncStatus],
+      queryFn: () => CloudSyncApi.fetchStatus(),
+    },
+  ];
+
+  return useQueries({
+    queries: QUERIES.map(query => query),
```

Comment:
Did you forget this map or is it just to clone the array ?

Reply:
Nice catch :)